code/__DEFINES/qdel.dm

```diff
@@ -24,9 +24,10 @@
 #define QDEL_HINT_IFFAIL_FINDREFERENCE 6
 #endif
 
-#define GC_QUEUE_CHECK 1
-#define GC_QUEUE_HARDDELETE 2
-#define GC_QUEUE_COUNT 2 //increase this when adding more steps.
+#define GC_QUEUE_FILTER 1
```

Comment:
Could you move these two define groups closer together and add some comments on purpose while you're adding this.